talent/hiring_process.md

```diff
@@ -0,0 +1,77 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+### Job Description
+
+- Job Description is written by the lead engineers with the Talent's Manager support. 
+
+- For every meeting between the lead and the Talent Manager there will be a PR in the [minutes](https://github.com/src-d/minutes) repo and the job descriptions will be uploaded at the private repository https://github.com/src-d/company.
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
+
+- The Talent Manager posts the job description through Lever on the source{d} website.
+
+### Sourcing
+
+- Lead engineers are responsible to identity people to be contacted and other sourcing channels such as niche job boards. 
+
+- At any given moment the Talent Manager is able to drop in and work on finding more developers.
+
+- Developer lists/sourcing channels should continuously be defined and refreshed after communication between the Talent Manager and the leads. 
+
+- The Talent Manager is responsible to work with the Developer Community team in order to gain people through community actions (Twitter/Blog etc.). 
+
+### First Contact
+
+- Any developer should be contacted by the lead engineers or under their name at least (after together defining the outgoing message). In case the Talent Manager is contacting them, it should be done from the lead's email acccount. It's a motivating factor for the developers to be contacted by the lead engineers. 
+
+### Screening
+
+- If the developer is an applicant, they should be accepted/rejected by the lead engineers during the screening phase. 
+
+- If the developer is approached by us from a specific source (known from meetups, having worked together in the past,  etc.), they should move forward to the next step as it means that the leads have already shown interest.
+
+### First Skype/Zoom/Phone Interview
+
+- A quick Skype/Phone/Zoom call should be available to every developer who is interested in the roles*. It is a general discussion to know more about the background and experience of the developer. And to share as much as possible from our side about the team & company. 
```

Comment:
The last sentence should be merged to the previous one, or simply rephrased into something else.